src/Client/ConfigWindow.java

```diff
@@ -941,50 +1018,66 @@ public void stateChanged(ChangeEvent e) {
     bicubicInterpolationScalingWarning.setAlignmentY(0.9f);
     bicubicInterpolationScalingWarning.setBorder(new EmptyBorder(0, 2, 0, 0));
     generalPanelBicubicScalingPanel.add(bicubicInterpolationScalingWarning);
+
+    configItems.add(
+        List.of(
```

Comment:
Method not present in Java 8. There are 23 usages of `List.of(` in this PR.

Reply:
Resolved; converted all usages to `Arrays.asList` instead which has the same signature.